src/ticketing/use-list-recurring-payments-query.tsx

```diff
@@ -0,0 +1,23 @@
+import {useAuthState} from '@atb/auth';
+import {listRecurringPayments} from '@atb/ticketing';
+import {useQuery} from '@tanstack/react-query';
+
+const ONE_HOUR_MS = 1000 * 60 * 60;
+
+export const LIST_RECURRING_PAYMENTS_QUERY_KEY = 'getListRecurringPayments';
+
+export const useListRecurringPaymentsQuery = () => {
+  const {authenticationType, abtCustomerId} = useAuthState();
+
+  return useQuery({
+    queryKey: [LIST_RECURRING_PAYMENTS_QUERY_KEY, abtCustomerId, authenticationType],
+    queryFn: async () => {
+      if (authenticationType !== 'phone') {
+        return undefined
+      }
```

Comment:
I just thought of something. Returning `undefined` for query function is not allowed. You haven't observed an error in Metro when returning `undefined`?

Reply:
Wait I'll double check it

Reply:
would this work instead? 

```
      if (authenticationType === 'phone') {
        return await listRecurringPayments();          
      }
```

So don't return anything if it is not a phone auth.

Reply:
Nope, that is the same as returning `undefined`.

Reply:
Oh, then would ´return []´ work? 🤔

Reply:
Changes pushed, thanks!